sw/host/opentitanlib/src/transport/cw310/usb.rs

```diff
@@ -175,6 +187,19 @@ impl Backend {
     pub const REQ_CS_HIGH: u16 = 0xA3;
     pub const REQ_SEND_DATA: u16 = 0xA4;
 
+    /// Commands for programming the bitstream into the FPGA.
+    pub const CMD_FPGA_STATUS: u8 = 0x15;
```

Comment:
nit: I am not sure if these are sm3u commands or if they are passed as is to the FPGA. A cross-reference to the cw310 would be nice to have.

Reply:
I'll try to write some documentation for the protocol, at least as well as I understand it from reading the chipwhisperer python implementation.